clearnode/rpc_router_auth.go

```diff
@@ -264,6 +264,13 @@ func (r *RPCRouter) handleAuthSigVerify(ctx context.Context, sig string, authPar
 		return nil, nil, "failed to create signer in db"
 	}
 
+	// Generate the User tag
+	_, err = GenerateOrRetrieveUserTag(r.DB, challenge.Address)
+	if err != nil {
```

Comment:
For brevity's sake, I'd go with:

```suggestion
	if _, err := GenerateOrRetrieveUserTag(r.DB, challenge.Address); err != nil {
```

Just a suggestion, definitely not a blocker 👍